smokes/e2e/reason_force.coffee

```diff
@@ -0,0 +1,88 @@
+# this file will contains the different generic functions which
+# will be called by the different tests
+#
```

Comment:
needs to add that this is inspired by this methodology

http://www.lindstromhenrik.com/using-protractor-with-coffeescript/